src/pages/AddServers.js

```diff
@@ -150,11 +151,20 @@ class AddServers extends BaseUpdateWizardPage {
   // values, then set size is the same as the array list length,
   // otherwise the list contains duplicate values.
   hasDuplicates = (arrayList) => {
-    return (new Set(arrayList)).size !== arrayList.length;
+    if(isEmpty(arrayList)) {
```

Comment:
This check is unnecessary. arrayList is the result of a map() function, so it will always be a valid array, and the rest of the logic works correctly when the array is empty

Reply:
that is true, will update